### ISSUE TITLE
Components / Breadcrumb - A11y fixes

### ISSUE THREAD

**Issue by yesenia-gtz** [Office-of-Digital-Services/California-State-Web-Template-Website#971]
[Components / Breadcrumb](https://as-cdt-pub-migr-betaws-ww-p-001-stage.azurewebsites.net/components/breadcrumb.html)

**Link text used for multiple different destinations**
The same link text is used for links going to different destinations. Users might not know the difference if they are not somehow explained.

- [ ] Use the same exact description for the text links and make sure they have the same URL destination
![Screen Shot 2022-12-13 at 1 03 41 PM](https://user-images.githubusercontent.com/98193284/207442989-c694f46d-e98f-4f05-b45d-4e41de2d7e14.png)
-------------

**Landmarks with identical names**
Two or more HTML5 or WAI-ARIA landmarks have identical names. Users might not know the difference if it's not somehow explained.

- [ ] Use unique names for each landmark on the page.

![Screen Shot 2022-12-13 at 1 04 21 PM](https://user-images.githubusercontent.com/98193284/207443099-bd661bcb-5095-47ed-8a0c-2233d1871049.png)
-------------
**Aria-atomic="true" missing on error container**
The page contains an error container made with the WAI-ARIA role="alert" or aria-live="assertive" attributes, but it does not contain a WAI-ARIA attribute that will make sure all assistive technologies can read the error message after more than one invalid submission.

- [ ] To make sure that all assistive technologies will read the error message after more than one invalid submission, you need to add the WAI-ARIA attribute aria-atomic="true" to the error container tag.

**Comment by kkoryaka** [comment]
@pglomboy please review. BTW, I also listed warnings here. Remember that we don't fix warnings, we only fix issues

**Comment by pglomboy** [comment]
perfect and fixed
